examples/twittermap/web/public/javascripts/timeseries/controllers.js

```diff
@@ -120,12 +125,14 @@ angular.module('cloudberry.timeseries', ['cloudberry.common'])
             $scope.queried = true;
             var ndx = $scope.ndx;
             if (ndx) {
-              ndx.remove();
-              ndx.add($scope.empty);
-              dc.redrawAll();
-              ndx.add(newVal);
-              dc.redrawAll();
-              return;
+              if (!onPlay) {
+                ndx.remove();
+                ndx.add($scope.empty);
+                dc.redrawAll();
+                ndx.add(newVal);
+                dc.redrawAll();
+              }
+                return;

```

Comment:
Please keep the indentation consistent. `return` should be the same level as `if` statement